src/main/python/hydra/lib/hydrabase.py

```diff
@@ -568,7 +554,7 @@ def reset_all_app_stats(self, name, group_name=""):
             ha_sub.reset_stats()
             ha_sub.stop()  # closes the ANalyser socket, can not be used anymore
 
-    def ping_all_app_inst(self, name, group_name=""):
+    def remove_unresponsive_tasks(self, name, group_name=""):
```

Comment:
Do a grep on the other repository that use hydra to make sure the changed function is not being used there.
i.e. do it on hCassandra/hZookeeper and hello world as well.